### PR TITLE
Improve the formatting of a formula in a .h file.

### PR DESCRIPTION
**Pull request by bangerth**
This doesn't change the output, just whether or not it's readable in the
source file. It isn't right now :-)

### CODE REVIEW

**Comment by bangerth** [comment]
I've pushed one more commit that clarifies what this class actually computes.

File: include/deal.II/numerics/error_estimator.h [hunk 1]
```diff
@@ -97,15 +97,30 @@ namespace hp
...
+ * functions of this class compute a vector of values that corresponds to
+ * $\eta_K$ (i.e., the square root of the quantity above).
+ *
+ * In the paper of Ainsworth $ c_F=\frac {h_K}{24} $, but this factor is a bit
  * esoteric, stemming from interpolation estimates and stability constants which
  * may hold for the Poisson problem, but may not hold for more general
- * situations. Alternatively, we consider the case when $ c_F=\frac {h_F}{2p_F}
- * $, where $ h_F $ is face diagonal and $ p_F=max(p^+,p^-) $ is the maximum
+ * situations. Alternatively, we consider the case when $c_F=\frac {h_F}{2p_F}$,
+ * where $h_F$ is face diagonal and $p_F=max(p^+,p^-)$ is the maximum
```
Comment (tjhei) [inline include/deal.II/numerics/error_estimator.h:123]:
```suggestion
 * where $h_F$ is the face diagonal and $p_F=max(p^+,p^-)$ is the maximum
```
?

Comment (bangerth) [reply include/deal.II/numerics/error_estimator.h]: Actually, it's the *diameter* of the face. I've fixed that.

File: include/deal.II/numerics/error_estimator.h [hunk 2]
```diff
@@ -97,15 +97,30 @@ namespace hp
...
+ * functions of this class compute a vector of values that corresponds to
+ * $\eta_K$ (i.e., the square root of the quantity above).
+ *
+ * In the paper of Ainsworth $ c_F=\frac {h_K}{24} $, but this factor is a bit
  * esoteric, stemming from interpolation estimates and stability constants which
  * may hold for the Poisson problem, but may not hold for more general
- * situations. Alternatively, we consider the case when $ c_F=\frac {h_F}{2p_F}
- * $, where $ h_F $ is face diagonal and $ p_F=max(p^+,p^-) $ is the maximum
+ * situations. Alternatively, we consider the case when $c_F=\frac {h_F}{2p_F}$,
+ * where $h_F$ is diameter of the face and $p_F=max(p^+,p^-)$ is the maximum
```
Comment (masterleinad) [inline include/deal.II/numerics/error_estimator.h:123]:
```suggestion
 * where $h_F$ is the diameter of the face and $p_F=max(p^+,p^-)$ is the maximum
```

**Comment by bangerth** [comment]
@masterleinad -- change made!

**Pull request merged**